dlls/client.cpp

```diff
@@ -271,7 +271,7 @@ int Q_UTF8ToUChar32( const char *pUTF8_, unsigned int &uValueOut, bool &bErrorOu
 	bErrorOut = true;
 	return nBytes;
 
-decodeFinishedMaybeCESU8:
+//decodeFinishedMaybeCESU8:
```

Comment:
May be comment all block then?
Or place under `#if 0`?

Reply:
yeah didn't notice it's pretty much dead code